.env.production.sample

```diff
@@ -45,6 +45,14 @@ ES_PASS=password
 SECRET_KEY_BASE=
 OTP_SECRET=
 
+# Encryption secrets
+# ------------------
+# Use `bin/rails db:encryption:init` to generate fresh secrets
```

Comment:
Possibly add some more information, that those are *secrets* that need to be common to all nodes of a Mastodon server, but not shared outside, and also that they should not be changed once set.

Reply:
Added some notes.